Fc 2082 3x payzen as a magento2 payment (#915)
front-commerce/developers.front-commerce.com: docs/03-extensions/payzen/how-to/magento2.mdx

---
title: Use as a Magento2 payment
description:
  "This guides explains how to configure Payzen payment with Magento 2 module."
sidebar_position: 4
---

<p>{frontMatter.description}</p>

:::caution Information

> <span>

Reviewer: **nit:** is the <span> block needed here?

  This integration is aimed at being transparent for administrators and
  developers. That is why we haven't duplicated documentation from existing
  Magento resources. Please <ContactLink /> if you need further assistance.
</span>

:::

Front-Commerce Magento2 module contains headless payment adapters for the
[**PayZen Standard** method](https://github.com/lyra/plugin-magento) (Lyra's
official Magento module).

The PayZen module must be configured in a normal way, as for a non-headless
Magento store.

## Pre-requisite

:::danger

For the time being, we only support PayZen accounts using a SHA1 signature. If
your account uses a SHA-256 signature, please contact support to learn about the
modifications required in the Front-Commerce Module to make it compatible.

:::

- [Payzen Account](https://www.payzen.eu/)
- [Payzen Module installed in Magento 2](#installation)

## Installation

:::danger PHP Module Compatibility

> At the time of writing, only version 2.5.13 of the Payzen module is supported.

Reviewer 2: **nitpick:** we do support previous versions and likely other patch versions (in case they're released)

**suggestion:**  rephrase
```suggestion
At the time of writing, we support the Payzen module up to version 2.5.
```

Author: fixed with your suggestion and added to 2.x docs

Support for next version will be available in the future.

:::

### Install module

Follow the
[official guide to setup Payzen in Magento](https://payzen.io/en-EN/plugins/magento2/prerequisites.html)

:::tip

If you're using composer, we recommend you to use it to install the module by
running the following command at the root of your Magento project:

```shell
composer config repositories.payzen git https://github.com/lyra/plugin-magento.git
composer require lyranetwork/module-payzen:2.5.13
```

:::

### Magento Patch

:::danger

This Patch update code in Magento dependency and may be overriten when updating
your PHP dependencies.

> We know that this is a dirty hack and we're looking for a cleaner method to
> prevent this patch.

Reviewer: ```suggestion
We know that this is a dirty hack and we're looking for a cleaner method to
prevent having to use this patch.
```

Author: fixed

:::

> In order for module to work properly you need to apply this patch to your

Reviewer: ```suggestion
In order for the module to work properly you need to apply this patch to your
```

Author: fixed
